app/src/main/java/com/mathewsachin/fategrandautomata/imaging/DroidCvPattern.kt

```diff
@@ -208,7 +212,7 @@ class DroidCvPattern(
     override fun threshold(value: Double): IPattern {
         val result = Mat()
 
-        Imgproc.threshold(mat, result, value * 255, 255.0, Imgproc.THRESH_BINARY)
+        Imgproc.threshold(mat, result, value * 255, 255.0, THRESH_BINARY)
```

Comment:
nitpick: revert back to `Imgproc.THRESH_BINARY` and remove the import. only used at 1 place now.

Reply:
Done